src/game/gamemodes/Passthebombv2.js

```diff
@@ -0,0 +1,198 @@
+      BOMB.graphic.addChild(this.bombtext);
+      this.bombtext.x = -100;
+      this.bombtext.y = -100;
+      Bombset = true;
+    }
+    if (Bombset) {
+      bombTimer -= dt;
+      this.bombtext.text = Math.ceil(bombTimer);
+      if (bombTimer < 0) {
+        BOMB.graphic.removeChild(this.bombtext);
+        BOMB.die();
+        Bombset = false;
+        BOMB = null;
+        this.time = 0;
+        bombTimer = 5;
+        bombExploded = true;
+      }
+    }
+  }
+
+  // Called after the game objects are updated.
+  postUpdate(dt) {
+    if (bombExploded) {
+      Object.keys(this.players).forEach(id => {
+        const entity = this.players[id];
+        if (!entity.dead) {
+          const score = this.game.scoreManager.getScore('Times Avoided Death', id);
+          this.game.scoreManager.setScore('Times Avoided Death', id, score + 1);
+        }
+      });
+      bombExploded = false;
+    }
+  }
+  // this.collision.setEntity(this);
+  //
+
+  // Called when a new player has been created
+  onPlayerCreated(playerObject, circle) {
+    const { iconID } = playerObject;
+    const idTag = playerObject.id;
+    circle.staticFriction = 0.8;
+    circle.collision.addListener((player, victim) => {
+      //      console.log(BOMB,player,victim);
+      if (player === BOMB && victim.isPlayer() && this.resttime > 0.5) {
+        BOMB.graphic.removeChild(this.bombtext);
+        BOMB = victim;
+        BOMB.graphic.addChild(this.bombtext);
+        this.resttime = 0;
+        bombTimer = Math.ceil(bombTimer);
+        const score = this.game.scoreManager.getScore('Bomb Passes', idTag);
+        this.game.scoreManager.setScore('Bomb Passes', idTag, score + 1);
+      } else if (victim === BOMB && this.resttime > 0.5) {
+        BOMB.graphic.removeChild(this.bombtext);
+        BOMB = player;
+        BOMB.graphic.addChild(this.bombtext);
+        this.resttime = 0;
+        bombTimer = Math.ceil(bombTimer);
+        const score = this.game.scoreManager.getScore('Bomb Passes', victim.controller.id);
+        this.game.scoreManager.setScore('Bomb Passes', victim.controller.id, score + 1);
+      }
+    });
+
+    // Place them in the middle of the arena for now
+    circle.x = this.arenaCenterx;
+    circle.y = this.arenaCentery;
+  }
+
+  // Called when a player disconnects
+  onPlayerLeave(idTag) {}
+
+  onButtonPressed(id, button) {}
+
+  /* eslint-enable class-methods-use-this, no-unused-vars */
+
+  // Clean up after the gamemode is finished.
+  cleanUp() {
+    this.arenaGraphic.destroy();
+  }
+
+  // Called when an entity is respawned.
+  onRespawn(entity) {
+    // Move the entity close to the center
+    entity.x = this.arenaCenterx + Math.cos(Math.random() * Math.PI * 2) * this.respawnArea;
+    entity.y = this.arenaCentery + Math.sin(Math.random() * Math.PI * 2) * this.respawnArea;
+  }
+
+  // Called when an entity dies.
+  // eslint-disable-next-line
+  onDeath(entity) {}
+
+  // eslint-disable-next-line
+  onWindowResize() {}
```

Comment:
Are you sure nothing needs to be done here? Thinking about position of rectangles.